config/quickstarthub/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
+  - ../ingress-nginx
```

Comment:
```suggestion
```
I'd argue this isn't needed for the quickstart as no Ingresses are created.
Would need to verify that.
If follow on guides need it (like something with ArgoCD or Thanos/Grafana), it can be added.
That being said, a Gateway should be running so a HttpRoute can be used instead of an Ingress then.